composer.json

```diff
@@ -138,7 +138,7 @@
   },
   "require-dev": {
     "ext-pgsql": "*",
-    "codeception/codeception": "2.4.0",
+    "codeception/codeception": "^2.4.0",
```

Comment:
Why was it 2.4.0 earlier? How was it resolved?

Reply:
i will add it into commit message after approval of changes